Mathlib/CategoryTheory/Comma/Final.lean

```diff
@@ -4,13 +4,15 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jakob von Raumer
 -/
 import Mathlib.CategoryTheory.Functor.KanExtension.Adjunction
-import Mathlib.CategoryTheory.Limits.Final
+import Mathlib.CategoryTheory.Limits.IsConnected
 import Mathlib.CategoryTheory.Grothendieck
 
 /-!
 # Finality of Projections in Comma Categories
 
 We show that `fst L R` is final if `R` is and that `snd L R` is initial if `L` is.
+As a corollary, we show that `Comma L R` with `L : A ⥤ T` and `R : B ⥤ T` is connected if `R` is
+final and `A` is connected.
```

Comment:
Could you add a `References` section to this module doc pointing at Prop. 3.4.3 of Kashiwara-Schapira?